scalableminds/webknossos
# Default to least recently used dashboard tab

### Mailable description of changes:
 - The dashboard remembers which tab was used most recently and defaults to that.

### Steps to test:
- Load dashboard, switch tabs, reload dashboard
- Ensure loading admin>user>show tracings still works and is unaffected by the tab switching

### Issues:
- contributes to #1938 

------
